### ISSUE TITLE
Executing ionic 2 serve throws Error: EPERM when windows 10 is started from the sleep state

### ISSUE THREAD

**Issue by akrammz** [ionic-team/ionic-cli#1248]
Executing ionic 2 serve throws Error: EPERM when windows 10 is started from the sleep state. Restarting the  laptop fixes the issue.

Here is the ionic version info
Cordova CLI: 6.1.1
Ionic Framework Version: 2.0.0-beta.10
Ionic CLI Version: 2.0.0-beta.35
Ionic App Lib Version: 2.0.0-beta.19
OS: Windows 8.1
Node Version: v0.12.7

Below are the errors after ionic serve is executed.
Running 'serve:before' gulp task before serve
[14:10:36] Starting 'clean'...
[14:10:36] Finished 'clean' after 20 ms
[14:10:36] Starting 'watch'...
[14:10:36] Starting 'sass'...
[14:10:36] Starting 'html'...
[14:10:36] Starting 'fonts'...
[14:10:36] Starting 'scripts'...
[14:10:36] 'html' errored after 29 ms
[14:10:36] Error: EPERM, mkdir 'C:\ionic2App\www\build'
    at Error (native)
[14:10:39] 'fonts' errored after 3.02 s
[14:10:39] Error: EPERM, mkdir 'C:\ionic2App\www\build\fonts'
    at Error (native)
Live Reload failed to start, error: Error: EPERM, operation not permitted 'C:\ionic2App\www\build'
Ionic serve failed - error: Error: EPERM, operation not permitted 'C:\ionic2App\www\build'
Ionic server commands, enter:
  restart or r to restart the client app from the root
  goto or g and a url to have the app navigate to the given url
  consolelogs or c to enable/disable console log output
  serverlogs or s to enable/disable server log output
  quit or q to shutdown the server and exit

ionic $ [14:10:40] 'sass' errored after 3.49 s
[14:10:40] Error: EPERM, mkdir 'C:\ionic2App\www\build\css'
    at Error (native)
[14:10:40] 'scripts' errored after 3.48 s
[14:10:40] Error: EPERM, mkdir 'C:\ionic2App\www\build\js'
    at Error (native)
8.3 MB bytes written (5.88 seconds)
Caught exception:
 Error: EPERM, mkdir 'C:\ionic2App\www\build\js'
    at Error (native)

**Comment by jgw96** [comment]
Hello @akrammz ! Are you running the latest version of Windows 10? Also would you mind running updating your ionic cli and testing again? Thanks for using Ionic!

**Comment by Mimicx** [comment]
i have the same issue...

Running 'serve:before' gulp task before serve
[18:28:07] Starting 'clean'...
[18:28:07] Finished 'clean' after 34 ms
[18:28:07] Starting 'watch'...
[18:28:07] Starting 'sass'...
[18:28:07] Starting 'html'...
[18:28:07] Starting 'fonts'...
[18:28:07] Starting 'scripts'...
[18:28:07] 'html' errored after 45 ms
[18:28:07] Error: EPERM: operation not permitted, mkdir 'D:\Applications\ess\www\bu
    at Error (native)
[18:28:11] 'scripts' errored after 3.43 s
[18:28:11] Error: EPERM: operation not permitted, mkdir 'D:\Applications\ess\www\bu
    at Error (native)
[18:28:11] 'fonts' errored after 3.44 s
[18:28:11] Error: EPERM: operation not permitted, mkdir 'D:\Applications\ess\www\bu
    at Error (native)
Live Reload failed to start, error: Error: EPERM: operation not permitted, lstat 'D
Ionic serve failed - error: Error: EPERM: operation not permitted, lstat 'D:\Applic
Ionic server commands, enter:
  restart or r to restart the client app from the root
  goto or g and a url to have the app navigate to the given url
  consolelogs or c to enable/disable console log output
  serverlogs or s to enable/disable server log output
  quit or q to shutdown the server and exit

ionic $ [18:28:11] 'sass' errored after 4.21 s
[18:28:11] Error: EPERM: operation not permitted, mkdir 'D:\Applications\ess\www\bu
    at Error (native)
9.1 MB bytes written (7.62 seconds)
Caught exception:
 Error: EPERM: operation not permitted, mkdir 'D:\Applications\ess\www\build\js'
    at Error (native)

**Comment by akrammz** [comment]
Hi @jgw96 
When I first reported the issue, I had the latest Windows 10 version but 'ionic info' output showed as Windows 8.1

I updated the ionic CLI, node, npm but still the same issue.
Here is the 'ionic info' output and this time it doesn't even display the OS version.

Cordova CLI: 6.1.1
Ionic Framework Version: 2.0.0-beta.10
Ionic CLI Version: 2.0.0-beta.37
Ionic App Lib Version: 2.0.0-beta.20
OS:
Node Version: v4.4.7


Thanks.
